paimon-core/src/main/java/org/apache/paimon/mergetree/LookupLevels.java

```diff
@@ -342,10 +345,10 @@ public byte[] persistToDisk(KeyValue kv, long rowPosition) {
                 segment.put(bytes.length - 1, kv.valueKind().toByteValue());
                 return bytes;
             } else {
-                byte[] bytes = new byte[8];
-                MemorySegment segment = MemorySegment.wrap(bytes);
-                segment.putLong(0, rowPosition);
-                return bytes;
+                // len: min 1 - max 9
```

Comment:
Maybe we can add two constants to VarLengthIntUtils to represent the min max of encodeLong